_includes/carpentries.html

```diff
@@ -10,11 +10,12 @@
   </div>
   <div class="col-md-8">
     <p><a href="{{ site.carpentries_site }}">The Carpentries</a> comprises
-    Software Carpentry and Data Carpentry, communities of Instructors, Trainers,
+    Software Carpentry, Data Carpentry, and Library Carpentry communities of Instructors, Trainers,
     Maintainers, helpers, and supporters who share a mission to teach
-    foundational coding and data science skills to researchers. In January,
+    foundational coding and data science skills to researchers and people 
+    working in library- and information-related roles. In January,
     2018, The Carpentries was formed by the merger of Software Carpentry and
-    Data Carpentry.</p>
+    Data Carpentry. Library Carpentry became an official Carpentries Lesson Program in November 2018.</p>
```

Comment:
shouldn't this be
> an official **Carpentry** Lesson Program

or

> Library Carpentry became an The Carpentries' official Lesson Program

Why do you capitalize "Lesson Program"?

Reply:
We capitalize words/concepts that have a specific meaning in our community (e.g., Instructors, Maintainers, ...)